### DISCUSSION TITLE
Which client to use for podgrab on Android?

### DISCUSSION THREAD

**Discussion by protonaut** [akhilrex/podgrab#291] [Q&A]
I just installed podgrab on my server and it works fine. I can reach podgrab through the internet using a proxy manager with an encrypted ssl  certificate (tlet's encrypt).
Within the Brwoser Podgrab works just fine.
I'm looking for a decent podcast-client working with podgrab for my android based mobile phone. 

I read that Antennapod works with podgrab. So I installed Antennapod and imported the feed I created on podgrab using the credentials. It seemed to work fine and I got a notification on my phone when new episodes were availible. Unfortunately I was not able to stream or download episodes trough Antennapod. I got an error massage suggesting that there was something wrong with the certifacate. I don't know if this is a problem of podgrab or not.

Is there any fix or android podcast clilent which works out of the box?

Thanks for any hints.

**Comment by cleverestx** [comment]
You ever find out?

**Reply by cnyrick** [reply]
Replying to an old post.....thought i would share, I was able to get it working, however i could not use my reverse proxy. i just subscribed to a podcast using http://<my-internal-ip>:8080/rss and it worked with Antennapod.  i was able to download/queue and play with no issues.
